### PR TITLE
refactor(condo): INFRA-155 rid off node env test usages

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

File: apps/address-service/index.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = {
                 getAddressProviderLimitHealthCheck(),
             ],
         }),
-        conf.NODE_ENV === 'test' ? undefined : new OIDCKeystoneApp(),
+        conf.DISABLE_OIDC_AUTH ? undefined : new OIDCKeystoneApp(),
```
Comment (nomerdvadcatpyat) [inline apps/address-service/index.js:66]: do we need add new envs in `.env.example` ?

**Review by nomerdvadcatpyat — APPROVED** [review]

File: run_condo_domain_tests.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ fi
 
 node bin/prepare.js -f condo -r condo
 
-export NODE_ENV=test
+export USE_LOCAL_FEATURE_FLAGS=true
```
Comment (Alllex202) [inline run_condo_domain_tests.sh:35]: Should we add `DISABLE_NEXT_APP =true`?

**Review by Alllex202 — APPROVED** [review]

File: run_condo_domain_tests.sh (hunk 2 above)
Comment (sitozzz) [reply run_condo_domain_tests.sh:35]: Yup, I forget to add it here

File: apps/address-service/index.js (hunk 1 above)
Comment (sitozzz) [reply apps/address-service/index.js:66]: I don't think so. This flag should be used in test environment only. But .env.example used to show how to startup project

**Pull request merged**